wfsim/core.py

```diff
@@ -433,15 +435,19 @@ def inverse_field_distortion(self, x, y, z):
         positions = np.array([x_obs, y_obs]).T 
         return z_obs, positions
 
-    def luminescence_timings(self, shape):
+    def luminescence_timings(self, xy, shape):
```

Comment:
Oops, didn't realize this line sneaked in with my commit. It's a bit tricky here, xy should a 2d array of [n_inst., 2], where the `shape` is [n_inst., n_photon] (the xy doesn't need to be the same for all inst.). So from here, the gas gap and the following pulse model need to be calculated for each instruction.
(I can also take care of this if @petergaemers can share the gas gap map?)

Reply:
Hi Tianyu, the gas gap map is on the private aux repo: https://github.com/XENONnT/private_nt_aux_files I did upload it to the wrong directory I just see